packages/blob-reader/helper.js

```diff
@@ -137,7 +122,7 @@ function diffMetaInformation(el) {
   const td = $(el).closest('td').get(0);
 
   // Blob view
-  if (td.classList.contains('js-file-line')) {
+  if (td.id.startsWith('LC') || isGist()) {
```

Comment:
This was the major change that GitHub did which caused most of the blob reader unit tests to fail. Previously `js-file-line` was only being used when looking at a single file, but now this class is always present even in diffs. The new approach checks the `id` attribute which starts with `LC` for single files and `diff-` in a diff scenario. Overall this exercise reminded me how complicated OctoLinker got over the past years.